catroid/src/main/java/org/catrobat/catroid/ui/SettingsActivity.java

```diff
@@ -401,10 +401,7 @@ public static void setAutoCrashReportingEnabled(Context context, boolean isEnabl
 		SharedPreferences.Editor editor = getSharedPreferences(context).edit();
 		editor.putBoolean(SETTINGS_CRASH_REPORTS, isEnabled);
 		editor.commit();
-
-		if (isEnabled) {
-			CrashReporter.initialize(context);
-		}
+		CrashReporter.initialize(context);
```

Comment:
`editor.putBoolean(SETTINGS_CRASH_REPORTS, isEnabled);` will determine if CrashReporter will initialize or not. I thought checking the same thing twice will be redundant. But yes, it makes sense that calling the function even when it is not required is odd. Ok, will change it 👍